torch/csrc/jit/codegen/cuda/kernel.cpp

```diff
@@ -197,17 +272,12 @@ void runKernel(
       1,
       0,
       stream,
-      arguments.data(),
+      kernel_arg_holder.args(),
```

Comment:
Didn't realize we could just pass a buffer to nvrtc. Can we check before hand if it's too big to throw a sensible error?

Reply:
I can do a rough check.
Just to make it clear. This is driver API (hence nothing to do with nvrtc). And we are passing an array of pointers (which points to the actual argument), driver will copy the pointed arguments.

Reply:
Ahh okay, misread. thanks.